etl/data_helpers/misc.py

```diff
@@ -152,3 +155,140 @@ def add_origins_to_mortality_database(tb_who: Table) -> Table:
 
 
 ########################################################################################################################
+
+
+def compare_tables(
```

Comment:
Having a shared module with QA functions, especially visualisations, is a great idea.